pallets/capacity/src/lib.rs

```diff
@@ -158,6 +164,21 @@ pub mod pallet {
 	pub type CapacityLedger<T: Config> =
 		StorageMap<_, Twox64Concat, MessageSourceId, CapacityDetails<BalanceOf<T>, T::BlockNumber>>;
 
+	/// Storage for the current epoch number
+	#[pallet::storage]
+	#[pallet::getter(fn get_current_epoch)]
+	pub type CurrentEpoch<T: Config> = StorageValue<_, T::BlockNumber, ValueQuery>;
```

Comment:
We may want to create a ticket to whitelist this so it does not cost a read when accessing it.

Reply:
I think we can also create a different PR for updating the fields where we are using block-number when we mean epoch number.